homeassistant/components/google_travel_time/sensor.py

```diff
@@ -206,6 +196,15 @@ def __init__(self, hass, config_entry, name, api_key, origin, destination):
             self.valid_api_connection = False
             return
 
+    async def async_added_to_hass(self) -> None:
+        """Handle when entity is added."""
+        if self.hass.state != CoreState.running:
+            self.hass.bus.async_listen_once(
+                EVENT_HOMEASSISTANT_START, lambda _: self.update
```

Comment:
I think we need to call update in the lambda, instead of just referencing the method.